pkg/controller/lifecycle/actuator.go

```diff
@@ -482,7 +469,17 @@ func getSeedResources(lakomReplicas *int32, namespace, genericKubeconfigName, sh
 
 	resources, err := registry.AddAllAndSerialize(
 		lakomDeployment,
-		lakomPDB,
+		&policyv1.PodDisruptionBudget{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      constants.ExtensionServiceName,
+				Namespace: namespace,
+				Labels:    getLabels(),
+			},
+			Spec: policyv1.PodDisruptionBudgetSpec{
+				MaxUnavailable: intstr.ValueOrDefault(nil, intstr.FromInt(1)),
```

Comment:
Nit: I would rather not wrap this in another function call. I find it easier and simpler to read

```golang
maxUnavailable = intstr.FromInt(1)
// and then 
MaxUnavailable: &maxUnavailable,
```

or directly 
```golang
 MaxUnavailable: &intstr.IntOrString{Type: intstr.Int, IntVal: 1}
```

Reply:
Done